egs/swbd/s5c/local/chain/tuning/run_tdnn_7m19c.sh

```diff
@@ -0,0 +1,383 @@
+#!/bin/bash
+# Note: before merging to master, this will be renamed.
+
+# 7m19c is as 7m19b but with one more layer (and moving the bypass connections up).
+#  Seems about 0.1% better.
+
+# local/chain/compare_wer_general.sh --rt03 tdnn7m19_sp tdnn7m19b_sp tdnn7m19c_sp
+# System                tdnn7m19_sp tdnn7m19b_sp tdnn7m19c_sp
+# WER on train_dev(tg)      13.09     12.93     12.86
```

Comment:
@GaofengCheng, I have now updated this branch to include example scripts, which are not finalized yet as I'm still tuning.
 This script is the one that's particularly impressive: thanks to the bottlenecks, this TDNN system on Switchboard is about 1% absolute better than the previous best TDNN system (1m).
I'm having trouble getting better results on the LSTM side though.  I may end up just recommending people to use TDNNs, which avoids all kinds of hassles anyway.

Reply:
great！applying SVD or something like inside gated recurrent unit needs sophisticated skills and exps. I will try this orth-layer on the TDNN layers of TDNN-LSTM and TDNN-OPGRU, OK?